Code/Framework/AzCore/Platform/Common/Apple/AzCore/std/parallel/internal/thread_Apple.cpp

```diff
@@ -70,7 +70,7 @@ namespace AZStd
         // QOS_CLASS_BACKGROUN         |           0          |           4          |  Will be prevented from using whole core.
         uint8_t GetDefaultThreadPriority()
         {
-            return 10;
+            return 25;
```

Comment:
So this value is somewhere between DEFAULT and USER_INITIATED?

Reply:
yeah the thread is now QOS_CLASS_DEFAULT instead of QOS_CLASS_UTILITY. Apple docs want you to use their api (i.e dispatch_queues) where you can specify the level and OS decides the priority. In our case we are using pthread so I am picking 25  to be in the middle.

Reply:
A comment explaining the rationale of picking 25 as the value would be nice. Otherwise, future code maintainers won't understand why to leave or change the value. Maybe something like:
```
// We currently set the thread priority somewhere in the middle of QOS_CLASS_DEFAULT by default, since our threads generally don't need to finish in a single frame. 
// Set this value higher if the default thread priorities are causing the threads to get starved too often.
```
As a side question, if this is for all our job manager threads, wouldn't USER_INITIATED be more appropriate? The description of "Asynchronous / cross-frame work" sounds more like what we use the job manager for by default than "streaming / multiple frames deadline".

Reply:
Yeah I did think that this could be an issue but didnt want to go higher as AP could in theory interfere with Editor usage. For now I am just jumping one level and if cpu threading becomes an issue we can jump to another level. Would be nice to pass in priority when creating threads (at higher level) as then we could set the correct one based on it's usage.